lib/saveInfo.js

```diff
@@ -1,18 +1,14 @@
-if (!localStorage.getItem("credentials")) {
-  localStorage.setItem("credentials", JSON.stringify([]));
-}
-
-const arr = JSON.parse(localStorage.getItem("credentials"));
-console.log(arr);
+function storeData() {
+  const newData = document.getElementById(".username").value;
```

Comment:
saying that the value is null but it should take in whatever I assign the data key to correct?

Reply:
Your syntax is invalid. Remove the '.' from username

Reply:
Did that and coded out the password input because it was throwing an error. But I still can't get the code data to local storage.